### DISCUSSION TITLE
I'm trying to make a mod for fun and I followed the tutorial on the fabric wiki and when I try to launch a world this is all I receive:

### DISCUSSION THREAD

**Discussion by AlexdaCat** [FabricMC/fabric-api#1842] [Mod Dev Support] [answered]
Errors in currently selected datapacks prevented the world from loading.
You can either try to load it with only the vanilla datapack ("safe mode"), or go back to the title screen and fix it manually.

There's no problems _in_ the mod (according to vscode at least).
Pressing Safe Mode does nothing.

**Comment by Daomephsta** [comment, marked as answer]
> There's no problems in the mod (according to vscode at least)

No IDE or IDE plugin can detect every possible issue. Something is wrong with your data files, check the game log.

**Reply by AlexdaCat** [reply]
found it! it was my pickaxe.json file - i had wrote <modname>_<blockname> instead of <modname>:<blockname>
